website/docs/d/logical_interface_profile.html.markdown

```diff
@@ -22,15 +31,15 @@ data "aci_logical_interface_profile" "example" {
 
 ## Argument Reference
 
-- `logical_node_profile_dn` - (Required) Distinguished name of parent Logical Node Profile object.
-- `name` - (Required) Name of Object logical interface profile.
+- `logical_node_profile_dn` - (Required) Distinguished name of the parent Logical Node Profile object.
+- `name` - (Required) Name of the object logical interface profile.
 
 ## Attribute Reference
 
 - `id` - Attribute id set to the Dn of the Logical Interface Profile.
-- `annotation` - (Optional) Annotation for object logical interface profile.
-- `description` - (Optional) Description for object logical interface profile.
-- `name_alias` - (Optional) Name alias for object logical interface profile.
+- `annotation` - (Optional) Annotation of the object logical interface profile.
+- `description` - (Optional) Description of the object logical interface profile.
+- `name_alias` - (Optional) Name alias of the object logical interface profile.
 - `prio` - (Optional) QoS priority class id.
 - `tag` - (Optional) Specifies the color of a policy label.
 
```

Comment:
We don't have the relationships in the data source?

Reply:
I'll add them in the next commit. I have to add the relationships in the data source first.